src/python/pants/engine/target.py

```diff
@@ -2292,7 +2292,7 @@ def generate_file_based_overrides_field_help_message(
         "overrides. You may either use a string for a single path / glob, "
         "or a string tuple for multiple paths / globs. Each override is a dictionary of "
         "field names to the overridden value.\n\n"
-        f"For example:\n\n{example}\n\n"
+        f"For example:\n\n```\n{example}\n```\n\n"
```

Comment:
Nit: maybe worth mentioning that the `example` argument is used in a code block?

Reply:
Like in docstring? I suspect most people will inspect the other uses of this function when they use it themselves

Reply:
Yea, in a docstring. Probably right, not a blocker.